### ISSUE TITLE
Undergraduate Machine Learning 28

### ISSUE THREAD

**Issue by Yajieniu** [Yajieniu/CMS-Progress-Mark#12]
## Neural Networks

1. Logistic regression: The logistic regression model specifies the probability of a binary output y -- y is either 0 or 1 (Bernoulli distribution)-- given an input x. 
    a) We assume a probability density of y is 3.14159265358979323846...(Pi)x. 
    b) The gradient of binary logistic regression looks like a measure of training errors. 
    c) H, the Hessian of the negative log-likelihood is positive definite. After getting it, we could implement Newton's method. We usually use Newton's method to solve linear regression.
2. Logistic regression with L1 regular rise is one of the most widely used techniques for binary classification. It's used in competitions all the time.
3. For a single neuron, we could calculate the probability of y_i=1 for each y_i (each training data point). And for n independent observations, the probability of all prediction values is the product of probabilities of all y's. 
3. For multi-neurons, we will have intermediate input and more complicated function. By designing more complex functions, we can break the linear boundary and create non-linear classifiers.
